scripts/codegen-firecracker.sh

```diff
@@ -23,12 +23,11 @@ echo
 
 # At the moment, we only test codegen for the virtio module
 cd $KANI_DIR/firecracker/src/devices/src/virtio/
-# Disable warnings until https://github.com/model-checking/kani/issues/573 is fixed
+export KANI_LOG=error
 export RUSTC_LOG=error
 export RUST_BACKTRACE=1
-# Use the legacy linker for now since we want to maximize the code that we are compiling from firecracker.
-# The MIR Linker will by default only collect code relevant to proof harnesses, however, firecracker has none.
-cargo kani --only-codegen --legacy-linker
+# Use cargo assess since this is now our default way of assessing Kani suitability to verify a crate.
```

Comment:
Not for this PR, but think we should do this for `std` too? (Or did we already and I missed it...)

Reply:
We haven't. I added a note to the description of the PR.

I don't know how easy it would be to compile the tests of the `std` library without the bootstrap script. We currently rely on the special cargo support to build the standard library. So even just building the standard library as part of `cargo kani` flow would require a way to extend our cargo call to include the `-Z build-std`.